packages/yew/src/virtual_dom/mod.rs

```diff
@@ -215,19 +216,45 @@ impl Attributes {
         // Remove missing
         for k in old.keys() {
             if !new.contains_key(k) {
-                Self::remove_attribute(el, k);
+                Self::remove_property_or_attribute(el, k);
             }
         }
     }
 
+    fn set_property(el: &Element, key: &str, value: &str) {
+        js_sys::Reflect::set(el.as_ref(), &JsValue::from_str(key), &JsValue::from_str(value)).expect("failed to set property");
+    }
+
     fn set_attribute(el: &Element, key: &str, value: &str) {
         el.set_attribute(key, value).expect("invalid attribute key")
     }
 
+    fn set_property_or_attribute(el: &Element, key: &str, value: &str) {
+        if el.has_own_property(&JsValue::from_str(key)) {
```

Comment:
This will always be false as properties are defined on the prototype of DOM elements :)
```suggestion
        if js_sys::Object::get_prototype_of(el).has_own_property(&JsValue::from_str(key)) {
```